README.md

```diff
@@ -253,19 +415,63 @@ Additionally, you must have a valid app in Nextmv Cloud.
 
 #### Push an application
 
-Place the following script in the root of your app directory and run it to push
-your app to the Nextmv Cloud. This is equivalent to using the Nextmv CLI and
-running `nextmv app push`.
+There are two strategies to push an application to the Nextmv Cloud:
 
-```python
-import os
+1. Specifying `app_dir`, which is the path to an app’s root directory. This
+   acts as an external strategy, where the app is composed of files in a
+   directory and those apps are packaged and pushed to Nextmv Cloud. This is
+   language-agnostic and can work for an app written in any language.
 
-from nextmv import cloud
+    Place the following script in the root of your app directory and run it to
+    push your app to the Nextmv Cloud. This is equivalent to using the Nextmv
+    CLI and running `nextmv app push`.
```

Comment:
Well, my intention for clarifying here is that people can rely on `nextmv-py` and not "know" about the CLI. I don’t know that the Python SDK should become a CLI tool as well, so I will defer that decision to others. In any case, that would  be a follow up and I would address that in a different PR.

Reply:
Fair point. I still think this is an odd or non-idiomatic experience for a non-notebook use-case (particularly for harmonic and CI/CD reasons). It would be nice to discuss this before we change the recommendation in general. :blush: